src/components/login/editProfile/PhoneAgreementView.js

```diff
@@ -20,18 +21,27 @@ const styles = StyleSheet.create({
   },
 });
 
-function PhoneAgreementView({ label }) {
+function PhoneAgreementView({ label, setError }) {
   const loginContext = useContext(LoginContext);
   const { user } = loginContext;
   const [agreement, setAgreement] = useState(
-    user.is_phone_visible === undefined ? true : user.is_phone_visible
+    user.is_phone_enabled === undefined ? true : user.is_phone_enabled
```

Comment:
I do not understand this line.

Reply:
The assumption was that it should work both with the current backend and after adding is_phone_visible field to the user. When I was creating it the user didn't have this field, so I assumed that by default this field is supposed to be true.  After updating the backend, the is_phone_visible field should be the same as in the database